tests/integration-tests/tests/update/test_update.py

```diff
@@ -412,6 +413,60 @@ def test_update_awsbatch(region, pcluster_config_reader, clusters_factory, test_
     _verify_initialization(region, cluster, updated_config)
 
 
+@pytest.mark.dimensions("eu-west-1", "c5.xlarge", "alinux2", "slurm")
+@pytest.mark.usefixtures("os", "instance")
+def test_update_compute_ami(region, pcluster_config_reader, clusters_factory, test_datadir):
+    # Create cluster with initial configuration
+    ec2 = boto3.client("ec2", region)
+    filters = [
+        {
+            "Name": "name",
+            "Values": ["aws-parallelcluster-" + get_installed_parallelcluster_version() + "-amzn2-hvm-x86_64*"],
+        }
+    ]
+    # Fixme choose private ami for develop branch vs public ami for release version
+    pcluster_ami_id = (
+        boto3.client("ec2").describe_images(ImageIds=[], Filters=filters, Owners=["self"]).get("Images")[0]["ImageId"]
+    )
+    init_config_file = pcluster_config_reader(global_custom_ami=pcluster_ami_id)
+    cluster = clusters_factory(init_config_file)
+    instances = cluster.instance()
+    logging.info(instances)
+    _check_instance_ami_id(ec2, instances, pcluster_ami_id)
+
+    # Update cluster with dlami as custom ami for compute queue
```

Comment:
this is not going to work during test of released version. To solve this, we should have a fixture which create a copy of the initial AMI, use it for the update and then delete on test termination.